plugins/core/IO/qStepCADImport/CMakeLists.txt

```diff
@@ -27,8 +30,8 @@ if ( PLUGIN_IO_QSTEP )
 	
 	set( OC_LIBRARIES TKSTEP TKSTEPBase TKSTEPAttr TKSTEP209 TKShHealing TKTopAlgo TKBRep TKGeomBase TKG3d TKG2d TKMath TKernel TKXSBase TKGeomAlgo TKMesh )
 	
-    target_link_directories( ${PROJECT_NAME} PRIVATE ${OPENCASCADE_LIB_DIR} )
-	target_link_libraries( ${PROJECT_NAME} ${OC_LIBRARIES} )
+    	target_link_directories( ${PROJECT_NAME} PRIVATE ${OPENCASCADE_LIB_DIR} )
```

Comment:
why this shift?

Reply:
To align the lines with the rest of cmake code

Reply:
Ah ok, this doesn't appear like this in github's interface 😅